WcaOnRails/config/routes.rb

```diff
@@ -145,7 +146,7 @@
   get 'wca-workbook-assistant' => 'static_pages#wca_workbook_assistant'
   get 'wca-workbook-assistant-versions' => 'static_pages#wca_workbook_assistant_versions'
   get 'organizer-guidelines' => 'static_pages#organizer_guidelines'
-  get 'tutorial' => redirect('/files/WCA_Competition_Tutorial.pdf', status: 302)
+  get 'tutorial' => redirect('/edudoc/competitor-tutorial/tutorial.pdf', status: 302)
```

Comment:
The Markdown version over in `wca-documents` is *NOT* feature-complete and a WIP. Please continue to use the static PDF from `files/` for the time being.